todo/serializers.py

```diff
@@ -114,6 +114,22 @@ class Meta:
         extra_kwargs = {
                         'owner': {'read_only': True}
                     }
-        
+    def to_representation(self, instance):
+        user = self.context['request'].user
+        if user.is_authenticated and instance.owner == user :
+            return super().to_representation(instance)
+        else:
+            return {'name' : instance.name}
 
+class TodoSerializerNotAuthenticated(serializers.ModelSerializer):
 
+    class Meta:
+        model = Todo
+        fields = ['name']
```

Comment:
```suggestion
        fields = ('name',)
```

ส่วนใหญ่ตรงนี้เราจะใช้เป็น tuple นะ แล้วก็ tuple เวลามี element เดียวจะต้องเติม comma ข้างหลัง